Sming/Makefile-rboot.mk

```diff
@@ -264,6 +264,7 @@ RBOOT_FW_BASE := $(abspath $(FW_BASE))
 export RBOOT_BIG_FLASH
 export RBOOT_BUILD_BASE
 export RBOOT_FW_BASE
+export RBOOT_RTC_ENABLED
```

Comment:
you might want to add a use of this (commented out by default) to the rBoot sample app makefile with the other options, so people will know it's there